src/models/participants.entity.ts

```diff
@@ -0,0 +1,19 @@
+import {DataType,Table,Column, Model} from "sequelize-typescript";
```

Comment:
nu cred ca ai nevoie de tabela asta ....citeste primele 20 de randuri de aici :https://sequelize.org/docs/v6/advanced-association-concepts/advanced-many-to-many/